person_capture/gui_app.py

```
@@ -11674,6 +11674,53 @@ def _try_save_wic_full_frame_crop_after_tonemap(
 
+            # Do not trust WIC-rendered pixels as the final dataset texture. The
+            # WIC/AVIF boundary is known to create colored salt pixels and dirty
+            # near-black texture. Keep its broad Paint-like tone/color as a
+            # reference, but write clean FFmpeg pixels corrected toward that
+            # low-frequency WIC look. If the clean render path fails, fall back to
+            # the historical WIC output so export reliability is unchanged.
+            try:
+                fx1, fy1, fx2, fy2 = [int(v) for v in final_crop]
+                wic_ref_crop = np.ascontiguousarray(bgr[fy1:fy2, fx1:fx2, :3], dtype=np.uint8)
+                guided_ok, guided_why, guided_changed = self._try_save_wic_guided_ffmpeg_crop(
+                    self._resolve_ffmpeg_bin() or ffmpeg_bin,
```

Ruff (0.15.12)

[error] Undefined name `ffmpeg_bin`

(F821)